frontend/package.json

```diff
@@ -0,0 +1,26 @@
+{
+    "name": "sample-app",
+    "version": "0.1.0",
+    "private": true,
+    "homepage": "./",
```

Comment:
`https://github.com/PlatformOfTrust/sample-app/`?

Reply:
It is used to build path for serving files.

Reply:
Well, that's not what the field is for

![image](https://user-images.githubusercontent.com/1296558/58003418-6612fd80-7ae9-11e9-9d25-eb59f17e4c91.png)

https://docs.npmjs.com/files/package.json#homepage

Reply:
Used by create react app in a different manner https://facebook.github.io/create-react-app/docs/deployment